common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1233,7 +1233,12 @@ def submit_problem(self, data, override_time=False):
             # without a stack trace
             else:
                 # Translators: {msg} will be replaced with a problem's error message.
```

Comment:
For extra credit: delete this comment. :) I know it isn't part of your changes, but someone left it behind a long time ago, and it's obsolete now.